test/ejabberd_sm_SUITE.erl

```diff
@@ -244,6 +263,27 @@ store_info_sends_message_to_the_session_owner(C) ->
             ct:fail("store_info_sends_message_to_the_session_owner=timeout")
     end.
 
+remove_info_sends_message_to_the_session_owner(C) ->
+    SID = {erlang:timestamp(), self()},
+    U = <<"alice2">>,
+    S = <<"localhost">>,
+    R = <<"res1">>,
+    Session = #session{sid = SID, usr = {U, S, R}, us = {U, S}, priority = 1, info = []},
+    %% Create session in one process
```

Comment:
remove, not store

Reply:
I think this comment was misplaced, right?